server/constants.py

```diff
@@ -85,6 +85,13 @@ class ExtraFileNames:
     """
     # Name for the tale config file
     tale_config = 'tale.yml'
+    license = 'license.txt'
```

Comment:
Does this mean that the exported or published Tale will have a file `license.txt` at the root?

Reply:
That's just the name of the file so that it's consistent throughout the project. It _is_ at the root of the package.

Reply:
Name changed to LICENSE in b50713c6763a93b19a15ed7d6ce6057fa5500db5